EIPS/eip-1679.md

```diff
@@ -24,50 +24,49 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
 
 ### Accepted EIPs
 
-- TBD
+- [EIP-1108](https://eips.ethereum.org/EIPS/eip-1108): Reduce alt_bn128 precompile gas costs
+- [EIP-1702](https://eips.ethereum.org/EIPS/eip-1702): Generalized account versioning scheme
+- [EIP-2024](https://ethereum-magicians.org/t/blake2b-f-precompile/3157): Blake2b Precompile
+- [EIP-2028](https://eips.ethereum.org/EIPS/eip-2028): Calldata gas cost reduction
 
 ### Rejected EIPs
 
-- TBD
+- [EIP-615](https://eips.ethereum.org/EIPS/eip-615): Subroutines and Static Jumps for the EVM - *Withdrawn*
+- [EIP-663](https://eips.ethereum.org/EIPS/eip-663): Unlimited SWAP and DUP instructions
+- [EIP-1109](https://eips.ethereum.org/EIPS/eip-1109): PRECOMPILEDCALL opcode (Remove CALL costs for precompiled contracts)
+  - requirement of EIP-1962
+- [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559): Fee market change for ETH 1.0 chain
+- [EIP-1930](https://eips.ethereum.org/EIPS/eip-1930): CALLs with strict gas semantic. Revert if not enough gas available.
+- [EIP-1959](https://eips.ethereum.org/EIPS/eip-1959): New Opcode to check if a chainID is part of the history of chainIDs
+- [EIP-1965](https://eips.ethereum.org/EIPS/eip-1965): Method to check if a chainID is valid at a specific block Number
+- [EIP-1985](https://eips.ethereum.org/EIPS/eip-1985): Sane limits for certain EVM parameters
+- [EIP-2014](https://eips.ethereum.org/EIPS/eip-2014): Extended State Oracle
+- [EIP-2025](https://eips.ethereum.org/EIPS/eip-2025): Funding Eth1.x with Developer Block Rewards *Withdrawn*
+- [EIP-2026](https://eips.ethereum.org/EIPS/eip-2026): State Rent H - Fixed Prepayment for accounts
```

Comment:
I think all the state rent ones were withdrawn?

Reply:
Just the one had a reference implementation. Unless that happened on the call after I dropped off.

Reply:
All but one were withdrawn